--- src/graphql/bondMarket.ts	2025-11-18 16:29:55
+++ src/graphql/bondMarket-new.ts	2025-11-18 16:30:57
@@ -267,7 +267,6 @@
   date: Scalars['String'];
   id: Scalars['ID'];
   market: Market;
-  marketId: Scalars['BigInt'];
   timestamp: Scalars['BigInt'];
 };
 
@@ -372,7 +371,6 @@
   Date = 'date',
   Id = 'id',
   Market = 'market',
-  MarketId = 'marketId',
   MarketBondContract = 'market__bondContract',
   MarketBondType = 'market__bondType',
   MarketCapacityInPayoutToken = 'market__capacityInPayoutToken',
@@ -407,7 +405,6 @@
   date: Scalars['String'];
   id: Scalars['ID'];
   market: Market;
-  marketId: Scalars['BigInt'];
   timestamp: Scalars['BigInt'];
 };
 
@@ -512,7 +509,6 @@
   Date = 'date',
   Id = 'id',
   Market = 'market',
-  MarketId = 'marketId',
   MarketBondContract = 'market__bondContract',
   MarketBondType = 'market__bondType',
   MarketCapacityInPayoutToken = 'market__capacityInPayoutToken',
